### DISCUSSION TITLE
Radio Group : How do I put the label text on the left close to the radio?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#3018] [Q&A] [answered]
Hi there

https://play.tailwindcss.com/TdxGkexGel

How can I ensure that "welding" and "Car Wash" are left aligned and are placed just next to the radio input?

![image](https://github.com/saadeghi/daisyui/assets/5799306/29cc3a16-16fd-48e8-bc78-ab5d89ca2e7e)

**Comment by saadeghi** [comment, marked as answer]
By default, `label`s have `space-between` style because that's the common thing to use.  
You can add `justify-start` to `label` to make the justify to the left:  
https://play.tailwindcss.com/nHcLMVBdTc

**Reply by Mashiane** [reply]
Thank you so much.
